pkg/service/service.go

```diff
@@ -116,7 +116,8 @@ func (svc *svcInfo) useMaglev() bool {
 	return option.Config.NodePortAlg == option.NodePortAlgMaglev &&
 		((svc.svcType == lb.SVCTypeNodePort && !isWildcardAddr(svc.frontend)) ||
 			svc.svcType == lb.SVCTypeExternalIPs ||
-			svc.svcType == lb.SVCTypeLoadBalancer)
+			svc.svcType == lb.SVCTypeLoadBalancer ||
+			(option.Config.ExternalClusterIP && svc.svcType == lb.SVCTypeClusterIP))
```

Comment:
Could you add a comment to explain why to provision when ClusterIP && ExternalClusterIP?